resources/js/pages/home/Components/HomePost.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { FaGift } from "react-icons/fa";
+import HomeSideProfile from './HomeSideProfile';
+import { FaRegSmile } from "react-icons/fa";
+import { FaTelegramPlane } from "react-icons/fa";
+import { FaFlag } from "react-icons/fa";
+
+
+
+export const HomePost = () => {
+    return (
+        <>
+            <div className="my-3 flex flex-col lg:flex-row">
+            <div className="hidden lg:block lg:w-1/4">
+                <HomeSideProfile />
```

Comment:
what does SideProfile and its parent of home page want inside the post component!?
they all belong to the home page component and in here must be the only logic related to the post itself